test/caesar_test.rb

```diff
@@ -1,2 +1,33 @@
+require './lib/caesar.rb'
+require 'minitest/autorun'
+require 'minitest/pride'
+require 'pry'
+
+class CaesarTest < Minitest::Test
+  def test_cipher_exists
+    cipher = CaesarCipher.new("the quick brown fox jumps over the lazy dog", 3)
```

Comment:
Since your class is called Cipher on line 1 in caesar.rb, you should be calling Cipher.new, not CaesarCipher.new.